include/sway/tree/view.h

```diff
@@ -95,6 +106,37 @@ struct sway_wl_shell_view {
 	int pending_width, pending_height;
 };
 
+struct sway_view_child;
+
+struct sway_view_child_impl {
+	void (*destroy)(struct sway_view_child *child);
+};
+
+/**
+ * A view child is a surface in the view tree, such as a subsurface or a popup.
```

Comment:
I'm so happy that popups are children of their parent windows now

Screw you wlc